core/dbt/events/types.py

```diff
@@ -2439,6 +2462,18 @@ def message(self) -> str:
         return str(self.exc)
 
 
+# TODO: new event
```

Comment:
This event seems duplicative with `NodeStart` + `PrintStartLine`. Is this new event needed, if we can support the same node details on those?

Reply:
In all honesty it was just a new event I added to start the work.  Then found `NodeStart`.  Have not taken the time to go back and see if it was needed - it's probably not.